docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -296,20 +283,20 @@ nodes` when run on the master.
 ### (Optional) Controlling your cluster from machines other than the master
 
 In order to get a kubectl on some other computer (e.g. laptop) to talk to your
-cluster, you need to copy the kubeconfig file from your master to your
-workstation like this:
+cluster, you need to copy the administrator kubeconfig file from your master
+to your workstation like this:
 
 ``` bash
 scp root@<master ip>:/etc/kubernetes/admin.conf .
```

Comment:
But still `{{ .KubeConfigPath}}` is `/etc/kubernetes/admin.conf`
So it's safest to always use that, not rely on a command that the user might have done
Personally I never copy it to `$HOME/.kube/config` but use `export KUBECONFIG=/etc/kubernetes/admin.conf`